src/dcp/dcp.c

```diff
@@ -79,6 +79,7 @@ void print_usage(void)
 #endif
     printf("  -b, --bufsize <SIZE>     - IO buffer size in bytes (default " MFU_BUFFER_SIZE_STR ")\n");
     printf("  -k, --chunksize <SIZE>   - work size per task in bytes (default " MFU_CHUNK_SIZE_STR ")\n");
+    printf("  -e, --xattrs <OPT>       - copy xattrs (none, all, non-lustre, libattr)\n");
```

Comment:
Do you mean to use ``-X`` now?
```suggestion
    printf("  -X, --xattrs <OPT>       - copy xattrs (none, all, non-lustre, libattr)\n");
```

Reply:
Yes